ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -237,6 +300,9 @@ public void tearDown() {
       vertx.close();
     }
     REST_APP.getServiceContext().close();
+
+    ((ConnectorList) makeKsqlRequest("SHOW CONNECTORS;").get(0)).getConnectors()
```

Comment:
How does this work if it's being called after `REST_APP.getServiceContext().close();`? 

Instead of trying to put this in `tearDown()`, we can wrap it into `givenConnectorExists()` (see comment above) and `classTearDown()` instead.